stor/tests/test_dx_path_compat.py

```diff
@@ -17,8 +17,12 @@ def test_construction_from_none(self):
             DXPath(None)
 
     def test_construction_from_no_project(self):
-        with pytest.raises(ValueError, match='Project is required to construct a DXPath'):
-            DXPath('dx://')
+        dx_p = DXPath('dx://')
+        self.assertEqual(dx_p, 'dx://')
+
+    def test_construction_no_project_fail(self):
+        with self.assertRaises(ValueError):
+            DXPath('dx:///folder')
```

Comment:
can you add a test with a value error on something like ```DXPath('dx://myproject/myfolder')``` just for clarity?

Reply:
Added